test.sh

```diff
@@ -3,10 +3,18 @@
 set -e
 echo "" > coverage.txt
 
+failed=0
+
 for d in $(go list ./... | grep -v vendor); do
-    go test -race -coverprofile=profile.out -covermode=atomic $d
+    go test -race -coverprofile=profile.out -covermode=atomic $d && true
```

Comment:
`&& true` is a trick to let script continue even a command failed with `set -e`.

Reply:
It preserves exit codes of commands (`$?`).